understanding/21/status-messages.html

```diff
@@ -17,7 +17,7 @@ <h2>Intent of this Success Criterion</h2>
 		<li> the message <q>provides information to the user on the success or results of an action, on the waiting state of an application, on the progress of a process, or on the existence of errors;</q></li>
 		<li> the message is not delivered via a change in context.</li>
 	</ol>
-<p>Information can be added to pages which does not meet the definition of a status message. For example, the list of results obtained from a search are not considered a status update and thus are not covered by this Success Criterion. However, brief text messages displayed about the completion or status of the search, such as "Searching...", "18 results returned" or "No results returned" would be status updates if they do not take focus. Examples of status messages are given in the section titled <a href="#status-examples">Status Message Examples</a> below.</p>
+<p>Information can be added to pages which does not meet the definition of a status message. For example, the list of results obtained from a search are not considered a status update and thus are not covered by this Success Criterion. However, brief text messages displayed <em>about</em> the completion or status of the search, such as "Searching...", "18 results returned" or "No results returned" would be status updates if they do not take focus. Examples of status messages are given in the section titled <a href="#status-examples">Status Message Examples</a> below.</p>
```

Comment:
Might be clearer changing "if they do not take focus" to "if they do not cause a change in context by taking focus"

Reply:
Note that only change in language here was  emphasis on "about". Fine if you want to make that change as long as you don't think it changes readability.